third-party/llvm/Makefile

```diff
@@ -12,6 +12,12 @@ else
   CHPL_LLVM_DEBUG := -DCMAKE_BUILD_TYPE=Release
 endif
 
+ifndef CHPL_LLVM_TARGETS_TO_BUILD
```

Comment:
It's pretty self-explanatory, but this might still warrant a comment for posterity